metrics/collectors/cnf/command_not_found.py

```diff
@@ -58,10 +58,19 @@ def get_cnf_ages_in_days(self):
                             date_str, "%Y-%m-%d %H:%M"
                         )
                         age = self.date_now - datetime_object
-                        age = age.total_seconds()
-                        data[s]["age (days)"] = age / 86400
+                        count = age.total_seconds() / 86400
             except urllib.error.HTTPError:
-                data[s]["age (days)"] = None
+                count = None
+            data.append(
+                {
+                    "measurement": "command_not_found_age",
+                    "fields": {
+                        "cnf_age": count,
+                        "out_of_date": ((count > 1) if count is not None else True),
```

Comment:
Setting out_of_date to True when there may have been a networking issue (HTTPError) feels incorrect to me. Additionally, if there is an HTTPError I think we should skip returning any data for that release series rather than writing what is effectively nothing to the database.

Reply:
Amended in the latest diff :)